chess/backend/src/game/migrations/0003_auto_20190606_1941.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2 on 2019-06-06 19:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
What does this line do?

Reply:
I don't know this was auto-generated by `python3 manage.py makemigrations`